src/migrations/1621511832113_ft-metadata.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable('ft_metadata', {
+    id: {
+      type: 'serial',
+      primaryKey: true,
+    },
+    name: {
+      type: 'string',
+      notNull: true,
+    },
+    description: {
+      type: 'string',
+      notNull: true,
+    },
+    image_uri: {
+      type: 'string',
+      notNull: true,
+    }, 
+    image_canonical_uri: {
+      type: 'string', 
+      notNull: true,
+    }, 
+    contract_id: {
+      type: 'string', 
+      notNull: true, 
+    },
+    symbol: {
+      type: 'string', 
+      notNull: true, 
+    },
+    decimals: {
+      type: 'integer', 
+      notNull: true, 
+    }
+    });
+
+    pgm.createIndex('ft_metadata', 'name');
+    pgm.createIndex('ft_metadata', 'description');
+    pgm.createIndex('ft_metadata', 'image_uri');
+    pgm.createIndex('ft_metadata', 'image_canonical_uri');
+    pgm.createIndex('ft_metadata', 'symbol');
+    pgm.createIndex('ft_metadata', 'decimals');
+    pgm.createIndex('ft_metadata', 'contract_id');
```

Comment:
Probably only index on `symbol`, `name`, and `contract_id`. We probably won't ever be querying by `description` for example.